zsh-proxy.plugin.zsh

```diff
@@ -186,9 +211,9 @@ __enable_proxy() {
         __enable_proxy_all
         echo "- git"
         __enable_proxy_git
-        echo "- npm & yarn"
+        # npm & yarn"
         __enable_proxy_npm
-        echo "- apt"
+        # apt"
```

Comment:
I see no necessary removing the output here.

Reply:
Has been moved into __enable_proxy_apt()